crypto/key.go

```diff
@@ -321,8 +322,24 @@ func (key *Key) GetFingerprint() string {
 	return hex.EncodeToString(key.entity.PrimaryKey.Fingerprint[:])
 }
 
+// GetSHA256Fingerprints gets the sha256 fingerprint from the key and subkeys
+func (key *Key) GetSHA256Fingerprints() (fingerprints []string) {
+	fingerprints = append(fingerprints, hex.EncodeToString(computeSHA256Fingerprints(key.entity.PrimaryKey)))
+	for _, sub := range key.entity.Subkeys {
+		fingerprints = append(fingerprints, hex.EncodeToString(computeSHA256Fingerprints(sub.PublicKey)))
+	}
+	return
+}
+
 // --- Internal methods
 
+// computeSHA256Fingerprints compute a fingerprint from a public key object
+func computeSHA256Fingerprints(pk *packet.PublicKey) []byte {
```

Comment:
This shouldn't be plural fingerprints - maybe
```suggestion
func getSHA256FingerprintBytes(pk *packet.PublicKey) []byte {
```
?